app/src/main/java/org/wikipedia/theme/Theme.java

```diff
@@ -68,4 +68,20 @@ public boolean isDark() {
         this.resourceId = resourceId;
         this.nameId = nameId;
     }
+
+    public static String getThemePageLibClass(Theme theme) {
+        switch (theme.getFunnelName()) {
+            case "light":
```

Comment:
Maybe it is okay to remove the `case "light"` since it is a default case?

Reply:
Done